curriculum/js1/9.js

```diff
@@ -4,8 +4,11 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, b = (a%7)) => {
+  if (b===0) {
+    return a+7;
+  }
+  return a+(7-b)
```

Comment:
* Please add a semicolon to the end of return statements
* This is a creative and unconventional solution. I had fun evaluating it. I understand that it will probably pass the tests as they're currently written. That said, what about the case where `a === -15`? The next multiple of 7 is -14. But this solution will return -7.

Reply:
Thanks for pointing out the problem with negative inputs. I'm sure there is a mathematical way to modify this while keeping the same logic, but it seemed simpler to just change the recursion a little bit. Could you please check?